### PR TITLE
[11.0] [MIG] partner_capital

### PR DESCRIPTION
**Pull request by remihb**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by remihb** [comment]
Does the 'oca-travis' commits have to be squash too ?

**Comment by pedrobaeza** [comment]
I think they are good now, thanks. Please check CLA part.

File: partner_capital/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Bla test
```
Comment (NL66278) [inline partner_capital/readme/DESCRIPTION.rst:1]: A few more details might be added here...

File: partner_capital/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
```
Comment (NL66278) [inline partner_capital/__manifest__.py:1]: You need to add the license key to this file to pass pylint checks.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: LGTM but one change is needed. Also consider to change license lines in headers from http to https. And please sign CLA if not already done.

**Comment by NL66278** [comment]
Requested changes not taken up, while travis failing.

**Pull request closed without merging**